packages/plugin-pnp/sources/PnpLinker.ts

```diff
@@ -278,6 +282,33 @@ export class PnpInstaller implements Installer {
       await xfs.chmodPromise(pnpDataPath, 0o644);
     }
 
+    if (pnpPath.main.endsWith(`.cjs`)) {
+      await xfs.writeFilePromise(pnpPath.esmLoader, `import { syncBuiltinESMExports, createRequire } from 'module';
+syncBuiltinESMExports();
+
+const pnpapi = createRequire(import.meta.url)('pnpapi');
+
+export async function resolve(specifier, context) {
+  const { parentURL = null } = context;
+
+  const resolvedPath = pnpapi.resolveRequest(
+    specifier.replace('file:///', ''),
```

Comment:
Isn't three slashes one too many on *NIX systems? e.g. `file:///Users/bram` is my homefolder, that'd change into `Users/bram` with this replacement

Node's `url` builtin has [`fileURLToPath `](https://nodejs.org/dist/latest-v14.x/docs/api/url.html#url_url_fileurltopath_url) and [`pathToFileURL`](https://nodejs.org/dist/latest-v14.x/docs/api/url.html#url_url_pathtofileurl_path) helpers.

Reply:
It is, switched to use those utils and fixed resolving builtins